pkg/defaults/mesh/signing_key.go

```diff
@@ -3,18 +3,28 @@ package mesh
 import (
 	"context"
 
+	"github.com/pkg/errors"
+
 	"github.com/kumahq/kuma/pkg/core/resources/manager"
 	core_store "github.com/kumahq/kuma/pkg/core/resources/store"
 	"github.com/kumahq/kuma/pkg/tokens/builtin/issuer"
 )
 
-func createSigningKey(resManager manager.ResourceManager, meshName string) error {
-	key, err := issuer.CreateSigningKey()
+func ensureSigningKey(resManager manager.ResourceManager, meshName string) (err error, created bool) {
```

Comment:
Isn't it much more familiar to put an error as the last parameter in return values? I'm not sure there is some style guide about it, just asking

Reply:
it should be the last one